test/libsolidity/semanticTests/viaYul/delete.sol

```diff
@@ -0,0 +1,44 @@
+contract C {
+	function internal_func() internal pure returns (int8)
+	{
+		return 1;
+	}
+	function call_internal_func() public pure returns (bool ret)
+	{
+		function() internal pure returns(int8) func = internal_func;
+
+		return func() == internal_func();
+	}
+	function call_deleted_internal_func() public pure returns (bool ret)
+	{
+		function() internal pure returns(int8) func = internal_func;
+
+		delete func;
+
+		return func() == internal_func();
+	}
+	function external_func() external pure returns (int8)
+	{
+		return 1;
+	}
+	/* TODO function() external is multi-slot and not yet implemented!
```

Comment:
Just remove it, I think we should anyway split up the tests more.

Also, the current SolidityEndToEndTests should have enough coverage, too.